src/app/page.tsx

```diff
@@ -1,9 +1,154 @@
+            quality={100}
+            layout="fill"
+            objectFit="cover"
+            fill
+          />
+          <Styles.ImageBackground />
+        </Styles.ImageContainer>
+        <Styles.TextContainer>
+          <Styles.Title>IMMIGRATION JUSTICE PROJECT</Styles.Title>
+          <Styles.Subtitle>
+            The Immigration Justice Project (IJP) seeks to promote due process
+            and access to justice through the provision of high quality legal
+            services on behalf of indigent immigrants and asylum seekers.
+          </Styles.Subtitle>
+          <Styles.ButtonDiv>
+            <LinkButton
+              $primaryColor={COLORS.goldMid}
+              $secondaryColor={COLORS.goldDark}
+              href="../signup"
+              style={{ fontSize: '1.5rem' }}
+            >
+              Learn More
+            </LinkButton>
+          </Styles.ButtonDiv>
+        </Styles.TextContainer>
+      </Styles.TitleSection>
+      <Styles.StatisticsSection>
+        {stats.map(stat => renderStat(stat))}
+      </Styles.StatisticsSection>
+      <Styles.HowYouCanHelpContainer style={{ background: COLORS.background }}>
+        <H2 $color={COLORS.blueMid}>
+          HOW <span style={{ color: COLORS.goldMid }}>YOU</span> CAN HELP
+        </H2>
+        <Styles.ServicesDiv>
+          {services.map(Service => renderService(Service))}
+        </Styles.ServicesDiv>
+      </Styles.HowYouCanHelpContainer>
+      <Styles.MissionValuesContainer>
+        <Styles.MissionStatement>
+          <H2 $color={COLORS.blueMid} style={{ maxWidth: '500px' }}>
+            OUR MISSION VALUES
+          </H2>
+          <H4 $fontWeight="400" style={{ maxWidth: '500px' }}>
+            The mission of the Immigration Justice Project (IJP) is to improve
+            public awareness of the legal system, to promote the administration
+            of justice, and to deliver high-quality legal services. IJP’s goals
+            are specifically targeted toward promoting due process and access to
+            justice at all levels of the immigration and appellate court systems
+            through the provision of high-quality legal services to individuals
+            navigating immigration legal proceedings in the San Diego,
+            California border region.
+          </H4>
+          <Styles.ButtonDiv>
+            <LinkButton
+              $primaryColor={COLORS.goldMid}
+              $secondaryColor={COLORS.goldDark}
+              href="https://www.americanbar.org/groups/public_interest/immigration/"
+            >
+              Learn More
+            </LinkButton>
+          </Styles.ButtonDiv>
+        </Styles.MissionStatement>
+        <Styles.ImageWrapper>
+          <Image
+            alt="Lawyer Image"
+            src={homepageImage.src}
+            placeholder="blur"
+            blurDataURL={homepageImage.src}
+            quality={100}
+            layout="fill"
+            objectFit="contain"
+          />
```

Comment:
similarly for this image: `objectFit="contain"` -> `style={{ objectFit: 'contain' }}`